src/Microsoft.Identity.Client/WsTrust/IWsTrustWebRequestManager.cs

```diff
@@ -66,5 +67,17 @@ internal interface IWsTrustWebRequestManager
             string userRealmUriPrefix,
             string userName,
             RequestContext requestContext);
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="userRealmUriPrefix"></param>
+        /// <param name="userName"></param>
+        /// <param name="requestContext"></param>
+        /// <returns>Userreal Discovery Endpoint</returns>
+        UriBuilder CreateUserrealmEndpoint(
```

Comment:
Since our endpoints are usually a Uri or string object, I'd rename this to:  CreateUserRealmEndpointUriBuilder()